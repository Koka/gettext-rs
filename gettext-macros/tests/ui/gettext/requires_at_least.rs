use gettext_macros::*;

fn main() {
    gettext!();
}
