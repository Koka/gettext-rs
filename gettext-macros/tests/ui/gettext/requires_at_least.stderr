error: requires at least a msgid string argument
 --> tests/ui/gettext/requires_at_least.rs:4:5
  |
4 |     gettext!();
  |     ^^^^^^^^^^
  |
  = note: this error originates in the macro `gettext` (in Nightly builds, run with -Z macro-backtrace for more info)
